src/DynamoCore/Configuration/PreferenceSettings.cs

```diff
@@ -462,12 +462,17 @@ public int NodeSearchTagSizeLimit
             set { NodeSearchTagSizeLimitValue = value; } 
         }
 
+        /// <summary>
+        /// The Version of the IronPython package that Dynamo will download when it is found as missing in graphs.
+        /// This static property is not serialized and is assigned IronPythonResolveTargetVersion's value 
+        /// if found at deserialize time.
+        /// </summary>
         internal static Version IronPythonResolveVersion = new Version(2, 1, 0);
 
         /// <summary>
         /// The Version of the IronPython package that Dynamo will download when it is found as missing in graphs.
         /// </summary>
-        public static string IronPythonResolveTargetVersion
+        public string IronPythonResolveTargetVersion
```

Comment:
I think this is an API break, why is this change necessary?

Reply:
It is a new API (just introduced it)
Nothing is going to break....

Reply:
so this is added to the preferences so that it can be set overridden by users?  
I am trying to figure out why this should be public/configurable.